src/test/java/api/support/matchers/EventMatcher.java

```diff
@@ -0,0 +1,71 @@
+package api.support.matchers;
+
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static org.folio.circulation.support.JsonPropertyFetcher.getBooleanProperty;
+import static org.hamcrest.core.Is.is;
+import org.hamcrest.Matcher;
+
+import org.folio.circulation.domain.representations.LoanProperties;
+import org.folio.circulation.services.EventPublishingService;
+
+import io.vertx.core.json.JsonObject;
+
+public class EventMatcher {
+
+  public static Matcher<JsonObject> isCheckedOutEvent() {
+    return JsonObjectMatcher.allOfPaths(
+      hasJsonPath("eventType", is(EventPublishingService.ITEM_CHECKED_OUT_EVENT_TYPE))
+    );
+  }
+
+  public static Matcher<JsonObject> isValidCheckedOutEventPayload(JsonObject loan) {
+    return JsonObjectMatcher.allOfPaths(
+      hasJsonPath("userId", is(loan.getString("userId"))),
+      hasJsonPath("loanId", is(loan.getString("id"))),
+      hasJsonPath("dueDate", is(loan.getString(LoanProperties.DUE_DATE)))
+    );
+  }
+
+  public static Matcher<JsonObject> isCheckedInEvent() {
+    return JsonObjectMatcher.allOfPaths(
+      hasJsonPath("eventType", is(EventPublishingService.ITEM_CHECKED_IN_EVENT_TYPE))
+    );
+  }
+
+  public static Matcher<JsonObject> isValidCheckedInEventPayload(JsonObject loan) {
+    return JsonObjectMatcher.allOfPaths(
+      hasJsonPath("userId", is(loan.getString("userId"))),
+      hasJsonPath("loanId", is(loan.getString("id"))),
+      hasJsonPath("returnDate", is(loan.getString(LoanProperties.RETURN_DATE)))
```

Comment:
Why do some of these checks use literals and the other uses a constant from the production code?

Reply:
Changed all to literals